backend/lcfs/web/api/fuel_supply/actions_service.py

```diff
@@ -113,6 +113,12 @@ async def _populate_fuel_supply_fields(
             else 0
         )
 
+        logger.error(fuel_supply.target_ci)
```

Comment:
remove

Reply:
Woops, thanks for catching that, removed!